lib/components/stepper/stepper.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./styles.scss";
+import locale from "../../locale";
+import { Heading } from "../heading";
+
+const getClassFromStepIfActive = (stepIndex: number, activeStep: number) =>
+  stepIndex === activeStep ? "mtfh-stepper__step--active" : "";
+
+interface StepperProps {
+  title?: string;
+  steps: string[];
+  activeStep?: number;
+}
+
+export const Stepper = ({ title, steps, activeStep = 0 }: StepperProps): JSX.Element => {
+  return (
+    <div>
+      {title && (
+        <Heading as="h2" variant="h2">
+          {title}
+        </Heading>
+      )}
+      <div className="mtfh-stepper mtfh-stepper--large mtfh-stepper--active">
+        <ol className="mtfh-stepper">
+          {steps.map((step, stepIndex) => (
+            <li
+              className={`mtfh-stepper__step ${getClassFromStepIfActive(
+                stepIndex,
+                activeStep,
+              )}`}
+              key={stepIndex}
+            >
+              <div className="mtfh-stepper__header">
```

Comment:
How do you feel about this:

```tsx
<Stepper>
    <Step>Tenure Details</Step>
    <Step>Search and Select</Step>
</Stepper>
```

I think this would also work really well as a Polymorphic component, like we have done in other components. What do you think?